kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -564,7 +564,7 @@ var ExerciseWrapperBaseView = BaseView.extend({
 
         this.current_attempt_log = new Models.AttemptLogModel(data);
 
-        this.attempt_collection.add(this.current_attempt_log);
+        this.attempt_collection.unshift(this.current_attempt_log);
```

Comment:
Attempt marks (X and checks) were being added to the lefthand side, but then showing back up on the right when refreshed. This fixes that.